### DISCUSSION TITLE
WM210V with ELS4 Basic and motors mount from Maxim

### DISCUSSION THREAD

**Discussion by Gixy31** [kachurovskiy/nanoels#15] [Show and tell]
Here is an implementation of the ELS4 Basic from Rocketronics, with the motors mounts from Maxim, with adaptations.
View of the controller mount with 3D printed support
![P5](https://user-images.githubusercontent.com/73664395/130976820-abb5e66b-a539-484c-9b76-a6628286b10f.jpg)
The electrical case, with motor drives, power supply and emergency stop button
![P1](https://user-images.githubusercontent.com/73664395/130976825-ae499220-3f84-4587-90b5-9aac12348aec.jpg)
The X motor, with a cable trail chain
![P2](https://user-images.githubusercontent.com/73664395/130976831-cea133a5-15d7-4293-bba3-0bde0ff72f1d.jpg)
The controller
![P3](https://user-images.githubusercontent.com/73664395/130976835-cb20b155-7364-4283-a74b-a5bd8edcfdfc.jpg)
The Z motor, the encoder gear (80T) and the differential adapter for the encoder signal
![P4](https://user-images.githubusercontent.com/73664395/130976838-0230b608-3215-48e5-b42d-56c2340a5561.jpg)

**Comment by kachurovskiy** [comment]
This looks awesome! Very nice modifications for the encoder gear. Which plastic did you use for printing?

**Comment by Gixy31** [comment]
Hello Maxim,
It's Prusament PETG, with 25% gyroid fill on a Prusa I3 MK3 printer. I've been obliged to put a screw on the 18T gear of the cross-slide, to avoid the gear slipping due to the motor torque. I think that I will do the same with the Z motor gear. WIth the torque and the temperature, the flat part isn't enough to maintain the gear and the hole becomes round.

**Reply by kachurovskiy** [reply]
Cool! BTW you can get aluminum HTD 5M gears on AliExpress for just a few dollars, they come with set screws and are very reliable.
